### ISSUE TITLE
SQL injection mitigation

### ISSUE THREAD

**Issue by ferboney** [WebGoat/WebGoat#548]
In this task you asked us for the IP to the "Webgoat-prd" server. After a bit of testing, i realised that all the IP adresses in the example worked. You can basically type in any IP that belongs to one of the servers, and this will be accepted. 

Best regards,

ferboney

**Comment by nbaars** [comment]
@ferboney correct, not the intended way to solve the lesson but there is no check on this field. We will need to add this check. Thanks!
